File: Mobile/GoodsTracker/app/src/main/java/com/example/jefferson/goodstracker/Communication/IDecoderFrame.java
package com.example.jefferson.goodstracker.Communication;

import com.example.jefferson.goodstracker.Domain.DataTelemetria;

/**
 * Created by Jefferson on 08/07/2017.
 */

public interface IDecoderFrame {

    boolean setValues(PayLoad payload, DataTelemetria b);
    boolean getValues(AnsCmd ans, DataFrame frame);
}

File: GoodsTracker/Communication/IDecoderFrame.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GoodsTracker
{
    interface IDecoderFrame
    {
        bool setValues(out PayLoad payload, Telemetria b);
        bool getValues(out AnsCmd ans, DataFrame frame);
    }
}
